res/templates/page.html

```diff
@@ -1,31 +1,25 @@
 <!DOCTYPE html>
 <html class='client-js'>
-
-<head>
-  <meta charset="UTF-8" />
-  <title></title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  __ARTICLE_CANONICAL_LINK__
-  __ARTICLE_CSS_LIST__
-  __CSS_LINKS__
-  __JS_SCRIPTS__
-</head>
-
-<body class="mediawiki mw-hide-empty-elt ns-0 ns-subject stable skin-minerva action-view animations">
-  <div id="mw-mf-viewport" class="feature-header-v2">
-    <div id="mw-mf-page-center">
-      <div id="content" class="mw-body">
-        <a id="top"></a>
-        <div id="bodyContent" class="content mw-parser-output">
-          <h1 id="titleHeading" style="background-color: white; margin: 0;"></h1>
-          <div id="mw-content-text">
+  <head>
+    <meta charset="UTF-8"/>
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/> __ARTICLE_CANONICAL_LINK__ __ARTICLE_CSS_LIST__
+    __CSS_LINKS__ __JS_SCRIPTS__ __CSS_LINKS_MOBILE__ __JS_SCRIPTS_MOBILE__ __PCS_CSS_OVERRIDE__
+  </head>
+  <body class="mediawiki mw-hide-empty-elt ns-0 ns-subject stable skin-minerva action-view animations">
+    <div id="mw-mf-viewport" class="feature-header-v2">
+      <div id="mw-mf-page-center">
+        <div id="content" class="mw-body">
+          <a id="top"></a>
+          <div id="bodyContent" class="content mw-parser-output">
+            <h1 id="titleHeading" style="background-color: white; margin: 0;"></h1>
+            <div id="mw-content-text"></div>
           </div>
         </div>
       </div>
     </div>
-  </div>
-  __ARTICLE_CONFIGVARS_LIST__
-  __ARTICLE_JS_LIST__
-</body>
-
+    __ARTICLE_CONFIGVARS_LIST__
+    __ARTICLE_JS_LIST__
+    __PCS_JS_OVERRIDE__
```

Comment:
If you need to override Mediawiki JS, then make a generic solution/naming to do that, but don't make it specific in particular with naming which is very Mediawiki specific (not MWoffliner

Reply:
Ok, I replaced it with MW_MOBILE naming.

Reply:
This is not what I meant. There should be only one place to override CSS and not a placeholder for each usage. A reall good name should be something like `__CSS_OVERRIDE__` or `__JS_OVERRIDE__` ... and inside you put whatever you want... here, for this end-point, the PCS related stuff.

Reply:
Done by creating a separate template for desktop and mobile.